### PR TITLE
docs: recommend jsdelivr to users and add download counts to the README

### PR DESCRIPTION
**Pull request by daviddias**
This PR:
- Adds download counts
- Recommend jsdelivr (faster CDN) and gives us the download counts

### CODE REVIEW

**Comment by achingbrain** [comment]
Interesting, what happened to unpkg?

**Comment by daviddias** [comment]
jsdelivr offers users a broader reach by leveraging multiple CDNs (it's quite impressive, see https://www.jsdelivr.com/network/infographic & https://www.jsdelivr.com/network)

**Comment by achingbrain** [comment]
Awesome.  Should something similar go into the readme for the http client too?

**Comment by achingbrain** [comment]
Wait, the http client change is being done in https://github.com/ipfs/js-ipfs/pull/2827 - why not do both changes in one PR?

**Comment by daviddias** [comment]
> why not do both changes in one PR?

Done, merged the other one into this PR :)

**Comment by daviddias** [comment]
Ack on the ask, @achingbrain. Unlikely that I can make until next week, fyi :)

**Comment by daviddias** [comment]
Done. 

The remaining unpkg references are not related to js-ipfs/js-ipfs-http-client
image

File: examples/browser-video-streaming/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <body>
     <video id="video" controls></video>
     <script src="./node_modules/ipfs/dist/index.js"></script>
-    <script src="https://unpkg.com/hlsjs-ipfs-loader@0.2.3/dist/index.js"></script>
```
Comment (achingbrain) [inline examples/browser-video-streaming/index.html]: Oops..

Comment (achingbrain) [reply examples/browser-video-streaming/index.html]: See https://github.com/ipfs/js-ipfs/pull/2914

**Pull request merged**